src/main/java/com/github/jnidzwetzki/bitfinex/v2/manager/OrderManager.java

```diff
@@ -326,7 +328,24 @@ public void cancelOrderGroup(final int id) throws BitfinexClientException {
 		}
 
 		logger.info("Cancel order group {}", id);
-		final CancelOrderGroupCommand cancelOrder = new CancelOrderGroupCommand(id);
+		final OrderCancelGroupCommand cancelOrder = new OrderCancelGroupCommand(id);
 		client.sendCommand(cancelOrder);
 	}
+
+    /**
+     * Cancel the given order group
+     * @throws BitfinexClientException
+     */
+    public void cancelAllOrders() throws BitfinexClientException {
+
+        final BitfinexApiKeyPermissions capabilities = client.getApiKeyPermissions();
+
+        if(! capabilities.isOrderWritePermission()) {
+            throw new BitfinexClientException("Unable to cancel all order - connection has not enough capabilities: " + capabilities);
```

Comment:
I found a small typo here:

```suggestion
            throw new BitfinexClientException("Unable to cancel all orders - connection has not enough capabilities: " + capabilities);
```